### ISSUE TITLE
Nord theme 😬

### ISSUE THREAD

**Issue by chriskrycho** [golf1052/base16-generator#7]
I'm back with more error reports for your excellent extension. 😁 Two issues with the Nord theme for you.

The Nord Light variation is… just broken. 😬 

<img width="1999" alt="light-theme" src="https://user-images.githubusercontent.com/2403023/31438723-50b4dc70-ae57-11e7-9285-712073bb3179.png">

The dark variation looks _great_… except for the status bar:

<img width="1888" alt="dark-status-bar" src="https://user-images.githubusercontent.com/2403023/31438780-9b20e722-ae57-11e7-9b21-361a56efde77.png">

**Comment by chriskrycho** [comment]
There are similar problems with contrast in the Cupertino themes.

**Comment by chriskrycho** [comment]
And with Mellow Purple!

<img width="1999" alt="mellow-purple-light" src="https://user-images.githubusercontent.com/2403023/31438909-2ad7a1f8-ae58-11e7-9afd-21bd752ad2e3.png">

**Comment by golf1052** [comment]
So I just checked out these themes on https://chriskempson.github.io/base16/ (well technically my updated fork -> https://github.com/golf1052/base16/tree/gh-pages) and for all three of these themes it looks like they just don't work as light themes.

The way I would prefer to solve this would be to block these three light themes from being generated. Since I'm pulling these themes from their respective repos I would rather defer to the theme creator to create a nice light version for their theme.

In the Cupertino case I would leave it as a light theme only and not generate the dark theme.

**Comment by chriskrycho** [comment]
Yeah, doing it the same (albeit inverted) as the GitHub theme seems right.

Note that it won’t solve the status bar problem on Nord Dark (which is actually what sent me poking around at the others).

**Comment by chriskrycho** [comment]
Note re:Nord, there's an official Nord repository upstream.

If you want, I can submit a PR over the weekend to remove the ones that look *really* bad.

**Comment by golf1052** [comment]
A PR would be great.

**Comment by Hedroed** [comment]
Hello
I made a fixed version of the nord dark theme.
![vscode_nord_dark](https://user-images.githubusercontent.com/17836466/66256671-8b4e9280-e790-11e9-8545-da426150ab74.png)

[base16-nord-dark.json.txt](https://github.com/golf1052/base16-generator/files/3693382/base16-nord-dark.json.txt)
Named `.txt` because github doesn't recognize `.json`.
